opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/MatchPhraseQueryTest.java

```diff
@@ -99,6 +109,26 @@ public void test_zero_terms_query_parameter() {
     Assertions.assertNotNull(matchPhraseQuery.build(new MatchPhraseExpression(arguments)));
   }
 
+  @Test
+  public void test_zero_terms_query_parameter_lowercase() {
+    List<Expression> arguments = List.of(
+        namedArgument("field", "t1"),
+        namedArgument("query", "t2"),
+        namedArgument("zero_terms_query", "all")
+    );
+    Assertions.assertNotNull(matchPhraseQuery.build(new MatchPhraseExpression(arguments)));
+  }
+
+  @Test
+  public void test_zero_terms_query_parameter_enum_random_case() {
+    List<Expression> arguments = List.of(
+        namedArgument("field", "t1"),
+        namedArgument("query", "t2"),
+        namedArgument("zEro_terMs_QUery", "ALL")
```

Comment:
what about changing case of `ALL`?

Reply:
Done in previous test case. Or do you mean doing both at the same time? There is no overlap of the code changes, so I didn't think it was needed to do both changes at the same time.

Reply:
That's fine.  We can skip.